frontend/src/utils/formatters.ts

```diff
@@ -37,3 +37,13 @@ export function toKebabCase(str: string | null) {
   const strs = str && str.match(/[A-Z]{2,}(?=[A-Z][a-z]+[0-9]*|\b)|[A-Z]?[a-z]+[0-9]*|[A-Z]|[0-9]+/g);
   return strs ? strs.join('-').toLocaleLowerCase() : '';
 }
+
+/**
+ * @function formatShortUuid
+ * Converts a UUID identifier to just show the first 8 characters 
+ * @param {String} value A typical UUID used from COMS
+ * @returns {String} The first 8 chars
+ */
+export function formatShortUuid(uuid: string) {
+  return uuid ? uuid.slice(0,8) : uuid;
```

Comment:
Wondering, as this is a common function now, can we have "The first 8 chars" to be parametrized?

Reply:
If that's the case, could just use `slice(0,8)` anywhere if the caller needs to specify length. I'd had the point of this one being to standardize the format anywhere some one calls it from (so then if that standard format changes the 8 would only be needed to be changed in one spot rather than from each caller).

If we want to have the param be callable (if there's reasons for variable format), I think probably just remove this function then

Reply:
Slice is a simple enough Array.prototype function I don't think we should be making a util function just for this, as it's part of the base language.

Reply:
Yeah util function was essentially a combo of the 8 constant and the way to do it. It's only used once at this point so can remove to the component.